Firestore/Protos/CMakeLists.txt

```diff
@@ -169,17 +170,18 @@ add_custom_target(
   VERBATIM
 
   COMMAND
+    cmake -E make_directory ${CMAKE_CURRENT_BINARY_DIR}/generated_protos
```

Comment:
Nice. But you should delete the line immediately below that this replaces.

Reply:
Done.